docs/en_US/multiPhase.md

```diff
@@ -40,4 +25,14 @@ To use multi-phase experiment, please follow below steps:
         # report final result somewhere for the parameter retrieved above
         nni.report_final_result()
         # ...
+    # ...
     ```
+To enable multi-phase, you should also add `multiPhase: true` in your experiment yaml configure file. If this line is not added, `nni.get_next_parameter()` would always return the same configuration. For all the built-in tuners/advisors, you can use multi-phase in your trial code without modification of tuner/advisor spec in the yaml configure file.
```

Comment:
it needs a title like below, or people may ignore the change.
__update experiment YAML configuration__

Reply:
updated